app/Traits/StoreImageTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Traits;
+
+use Storage;
+use Illuminate\Support\Str;
+
+trait StoreImageTrait
+{
+    public function uploadImageToStorage($file, $foderName)
+    {
+        $filePath = Storage::putFile('public/' . $foderName, $file);
```

Comment:
use `disk()` : https://laravel.com/docs/8.x/filesystem#the-local-driver
m suggest lưu data và storage và dùng storage link để truy xuất qua public

Reply:
vâng em dùng storage link nhưng lúc lưu không thêm phần  "pulbic/" vào trong hàm thì nó không lưu được vào storage ạ